azurelinuxagent/common/protocol/migration_util.py

```diff
@@ -0,0 +1,69 @@
+# Microsoft Azure Linux Agent
+#
+# Copyright 2020 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
+#
+
+import os
+import azurelinuxagent.common.conf as conf
+
+from azurelinuxagent.common.protocol.goal_state import TRANSPORT_CERT_FILE_NAME
+from azurelinuxagent.common.protocol.util import WIRE_PROTOCOL_NAME
+from azurelinuxagent.common.utils.restutil import KNOWN_WIRESERVER_IP
+
+# MetadataServer Legacy Certificates for Cleanup
+LEGACY_TRANSPORT_PRV_FILE_NAME = "V2TransportPrivate.pem"
```

Comment:
I would name those METADATA somewhere on on those names, and would probably rename the file to metadata_migration_util.